src/main/java/com/delphix/masking/initializer/maskingApi/SetupDriver.java

```diff
@@ -79,6 +79,7 @@
 
 public class SetupDriver {
 
+    protected static final String DATABASE_TYPE_EXTENDED = "EXTENDED_DATABASE";
```

Comment:
I think SetupDriver class is just being used in com.delphix.masking.initializer.main.java to call the run method, So can we make DATABASE_TYPE_EXTENDED property as a private instead of protected.

Reply:
Also I think the value should be **EXTENDED**
ie.
**private static final String DATABASE_TYPE_EXTENDED = "EXTENDED";**

Reply:
Fixed.